kubernetes/nodecollection.go

```diff
@@ -429,7 +430,12 @@ func allowDirectConnect(config KubeAgentConfig, nodes []v1.Node) bool {
 	// if any Fargate nodes are found.
 	for _, n := range nodes {
 		if isFargateNode(n) {
-			log.Infof("Fargate node found in cluster, direct node connection disabled. Learn more about Fargate support: %s", kbURL) //nolint: lll
+			log.Infof("Direct connection to an AWS Fargate node is not possible, so when a Fargate node is " +
+				"detected in the cluster, the agent switches to using kube-proxy for all node connections " +
+				"in the cluster. The allocation of EKS Fargate tasks are not currently supported via the " +
+				"containers feature dashboard. " +
+				"Please contact your technical account manager for instructions on how to access" +
+				"EKS Fargate information via the reporting feature.") //nolint: lll
```

Comment:
Similar to above, what do you think about changing these to a back-ticked, multiline string? (And I think that would get rid of the `nolint` too)